From fc20e1afc4a8915220859d0e09b411bb413cbfb5 Mon Sep 17 00:00:00 2001
Message-ID: <fc20e1afc4a8915220859d0e09b411bb413cbfb5.1728658722.git.sharlatanus@gmail.com>
In-Reply-To: <cover.1728658722.git.sharlatanus@gmail.com>
References: <cover.1728658722.git.sharlatanus@gmail.com>
From: Sharlatan Hellseher <sharlatanus@gmail.com>
Date: Sat, 5 Oct 2024 21:02:13 +0100
Subject: [PATCH 1/2] gnu: Add packages/python-graphics module.

* gnu/packages/python-graphics.scm: New file.
* gnu/local.mk: Add gnu/packages/python-graphics.scm to the scope.

Change-Id: I1d077c838ff1c3c014c3d699b539eaf8befd5084
---
 gnu/local.mk                     |  1 +
 gnu/packages/python-graphics.scm | 28 ++++++++++++++++++++++++++++
 2 files changed, 29 insertions(+)
 create mode 100644 gnu/packages/python-graphics.scm

diff --git a/gnu/local.mk b/gnu/local.mk
index afff609501..b47d4dc917 100644
--- a/gnu/local.mk
+++ b/gnu/local.mk
@@ -561,6 +561,7 @@ GNU_SYSTEM_MODULES =				\
   %D%/packages/python-check.scm			\
   %D%/packages/python-compression.scm		\
   %D%/packages/python-crypto.scm		\
+  %D%/packages/python-graphics.scm			\
   %D%/packages/python-science.scm		\
   %D%/packages/python-web.scm			\
   %D%/packages/python-xyz.scm			\
diff --git a/gnu/packages/python-graphics.scm b/gnu/packages/python-graphics.scm
new file mode 100644
index 0000000000..ef981bd8d7
--- /dev/null
+++ b/gnu/packages/python-graphics.scm
@@ -0,0 +1,28 @@
+;;; GNU Guix --- Functional package management for GNU
+
+(define-module (gnu packages python-graphics)
+  #:use-module ((guix licenses) #:prefix license:)
+  #:use-module ((guix build-system python) #:select (pypi-uri))
+  #:use-module (guix build-system pyproject)
+  #:use-module (guix download)
+  #:use-module (guix gexp)
+  #:use-module (guix packages)
+  #:use-module (gnu packages)
+  #:use-module (gnu packages check)
+  #:use-module (gnu packages python-build)
+  #:use-module (gnu packages python-xyz))
+
+;;; Commentary:
+;;;
+;;; Python modules which are meant to be used in GUI creation or mainly are
+;;; bindings to low level libraries such as Glue, Mesa, OpenGL, Xorg etc.
+;;;
+;;; Please: Try to add new module packages in alphabetic order.
+;;;
+;;; Code:
+
+
+;;;
+;;; Avoid adding new packages to the end of this file. To reduce the chances
+;;; of a merge conflict, place them above in alphabetical order.
+;;;
-- 
2.46.0

